msbuild/Xamarin.MacDev.Tasks/Tasks/ACToolTaskBase.cs

```diff
@@ -87,6 +89,12 @@ protected override void AppendCommandLineArguments (IDictionary<string, string>
 				}
 			}
 
+			if (!string.IsNullOrEmpty (AccentColor)) {
+				args.Add ("--accent-color", AccentColor);
+			} else {
+				args.Add ("--accent-color", "AccentColor");
+			}
```

Comment:
There doesn't seem to be a way to not pass --accent-color if it turns out problematic for some reason, so maybe it would make more sense to not pass anything if the AccentColor property isn't set? In that case, people can always get the default behavior by setting AccentColor="AccentColor".

Reply:
Going by the man file (https://keith.github.io/xcode-man-pages/actool.1.html#accent-color) if the color is invalid, a warning is generated but doesn't break the build. The system default is used instead.

Reply:
Yeah, but I don't trust Apple to not break their own stuff... and they could also add new features in the future that would require *not* passing `--accent-color`.

In any case my main concern is that with this PR we're going from never passing `--accent-`color to always passing it, and without an escape hatch if something is wrong, which seems quite risky to me.

Reply:
Good point. Removing it is fine with me since it will default to what it is doing now. Since you need to add it to the Asset Catalog anyway, and the default name is not obvious unless you know Xcode, removing it and documenting somewhere that it's a value you can set seems good to me.